src/content/docs/en/pages/devtools/cli/azion-cli-alpha/dev-command/dev.mdx

```diff
@@ -9,6 +9,10 @@ permalink: /documentation/products/cli/dev-command/
 
 import Button from '~/components/Button.astro'
 
+:::note 
+In case one or more required fields aren't informed through the specific flags, an interactive message prompts and asks for the missing information.
```

Comment:
```suggestion
In case any required field isn't informed through the specific flags, an interactive message prompts and asks for the missing information.
```